layouts/dashboard.vue

```diff
@@ -1,59 +1,78 @@
 <template>
     <div>
-        <app-header />
-        <dashboard-header />
+        <AppHeader />
+        <DashboardHeader />
         <Nuxt />
-        <kyb-alert v-if="showKybAlert" />
-        <kyc-alert v-if="showKycAlert" />
-        <b-alert
+        <KYBAlert v-if="showKybAlert" />
+        <KYCAlert v-if="showKycAlert" />
+        <BAlert
             id="verify-mobile"
             :show="showVerifyMobileAlert && !showVerifyEmailAlert"
             class="fixed-bottom bottom-left-alert m-4 p-4"
             variant="bg-colored"
         >
             We need to verify your mobile number. Please click
             <b-link class="link" to="/login/verify/mobile"> here.</b-link>
-        </b-alert>
-        <b-alert
+        </BAlert>
+        <BAlert
             id="verify-email"
             :show="showVerifyEmailAlert"
             class="fixed-bottom bottom-left-alert m-4 p-4"
             variant="bg-colored"
         >
             We need to verify your email address. Please click
             <b-button class="link mb-1" variant="transparent" @click="goToVerify">here.</b-button>
-        </b-alert>
+        </BAlert>
         <LoadingSpinner id="loader" :is-loading="isLoading" />
-        <cookie-policy />
+        <Cookie />
     </div>
 </template>
 
 <script lang="ts">
-import { Component, mixins } from 'nuxt-property-decorator'
-import KyVerified from '~/mixins/kyVerified'
-import BaseMixin from '~/mixins/BaseMixin'
+import { defineComponent } from '@nuxtjs/composition-api'
+import { computed } from 'vue'
+import AppHeader from '~/components/Header.vue'
+import Cookie from '~/components/cookie.vue'
 import LoadingSpinner from '~/components/atoms/LoadingSpinner.vue'
+import KYBAlert from '~/components/corporates/KYBAlert.vue'
+import KYCAlert from '~/components/consumers/KYCAlert.vue'
+import DashboardHeader from '~/components/DashboardHeader.vue'
+import { useStores } from '~/composables/useStores'
+import { useKyVerified } from '~/composables/useKyVerified'
+import { useBase } from '~/composables/useBase'
 
-@Component({
+export default defineComponent({
     components: {
+        Cookie,
+        DashboardHeader,
+        KYCAlert,
+        KYBAlert,
         LoadingSpinner,
-        AppHeader: () => import('~/components/Header.vue'),
-        DashboardHeader: () => import('~/components/DashboardHeader.vue'),
-        KybAlert: () => import('~/components/corporates/KYBAlert.vue'),
-        KycAlert: () => import('~/components/consumers/KYCAlert.vue'),
-        cookiePolicy: () => import('~/components/cookie.vue'),
+        AppHeader,
     },
-})
-export default class DefaultLayout extends mixins(KyVerified, BaseMixin) {
-    get isLoading() {
-        return this.loaderStore.isLoading
-    }
+    setup() {
+        const { loader } = useStores(['loader'])
+        const { accounts } = useStores(['accounts'])
```

Comment:
Can be done in one line

Reply:
done in fact no need for account - removed accounts